### PR TITLE
Changes to rename dotnet files for tests

### PR DESCRIPTION
**Pull request by alerickson**

### CODE REVIEW

File: Tests/PSGetPublishModule.Tests.ps1 [hunk 1]
```diff
@@ -1027,6 +1036,283 @@ Describe PowerShell.PSGet.PublishModuleTests -Tags 'BVT','InnerLoop' {
...
+                $err = $_
+            }
+
+            AssertNull $err "$err"
+            AssertNull $result "$result"
+            Assert (test-path $script:ProgramDataExePath) "NuGet.exe did not install properly"
+            AssertNotEquals (Get-Command $script:ProgramDataExePath).FileVersionInfo.FileVersion $oldNuGetExeVersion "Incorrect version of NuGet.exe"
+            
+            $module = find-module $script:PublishModuleName -RequiredVersion $version
+            AssertEquals $module.Name $script:PublishModuleName "Module published when it should not have"
```
Comment (bmanikm) [inline Tests/PSGetPublishModule.Tests.ps1]: >"Module published when it should not have" [](start = 64, length = 42)

Module should be published, right? #Closed

File: Tests/PSGetPublishModule.Tests.ps1 [hunk 2]
```diff
@@ -1027,6 +1036,283 @@ Describe PowerShell.PSGet.PublishModuleTests -Tags 'BVT','InnerLoop' {
...
+                RemoveItem $outputFilePath
+            }
+
+            AssertNull $result "$result"
+            Assert (test-path $script:ProgramDataExePath) "NuGet.exe did not install properly"
+            AssertNotEquals (Get-Command $script:ProgramDataExePath).FileVersionInfo.FileVersion $oldNuGetExeVersion "Incorrect version of NuGet.exe"
+            Assert ($content -and ($content -match 'upgrade')) "Publish module confirm prompt is not working, $content"
+
+            $module = find-module $script:PublishModuleName -RequiredVersion $version
+            Assert ($module.Name -eq $script:PublishModuleName) "Module published when it should not have"
```
Comment (bmanikm) [inline Tests/PSGetPublishModule.Tests.ps1]: >"Module published when it should not have" [](start = 64, length = 42)

Same here #Closed

File: Tests/PSGetPublishModule.Tests.ps1 (hunk 1 above)
Comment (alerickson) [reply Tests/PSGetPublishModule.Tests.ps1]: Yes that's right, I just updated that (in all applicable areas) and pushed #Closed

**Review by bmanikm — APPROVED** [review]
:shipit:

**Pull request merged**